tools/docker-compose/ansible/roles/sources/templates/local_settings.py.j2

```diff
@@ -20,6 +20,19 @@
 # INTERNAL_IPS = ('172.19.0.1', '172.18.0.1', '192.168.100.1')
 # ALLOWED_HOSTS = ['*']
 
+
```

Comment:
Along those same lines, this file still has a lot of things in it that are based around the assumption that it is going to be used as a developer's local settings overrides, with values to be commented and uncommented as circumstances warrant.